zkstack_cli/crates/zkstack/src/commands/ecosystem/setup_observability.rs

```diff
@@ -20,7 +20,16 @@ pub fn run(shell: &Shell) -> anyhow::Result<()> {
         ERA_OBSERBAVILITY_GIT_REPO,
         ERA_OBSERBAVILITY_DIR,
     )?;
-    spinner.finish();
 
+    // Add lambda remote and checkout to `eigenda` for DA metrics
+    git::add_remote(
+        shell,
+        path_to_era_observability.clone(),
+        "lambda",
+        "https://github.com/lambdaclass/era-observability.git",
+    )?;
+    git::checkout(shell, path_to_era_observability.clone(), "eigenda")?;
+
+    spinner.finish();
```

Comment:
I think the idea is to not do this, that is why we have a PR to era-observability

Reply:
If that's the case then an temporary step should be added to the instructions, specifying how to checkout to the needed branch while said PR remains unmerged.

Reply:
Let's add that and remove that code then!

Reply:
Done!